### DISCUSSION TITLE
Why `: void` is not used in test cases?

### DISCUSSION THREAD

**Discussion by datamweb** [codeigniter4/shield#197] [Q&A] [answered]
@kenjis I usually read the tests written by you. You usually wrote the test in Figure :
`public function testYourTest(): void
    {
// test code
    }
`
I know this is optional, but it's weird why you didn't do it!
May I ask why you did not use `void` here?
https://github.com/codeigniter4/shield/blob/3a7894c7fe51cefc7a943973f918acd661763248/tests/Authentication/Authenticators/AccessTokenAuthenticatorTest.php#L51

**Comment by kenjis** [comment, marked as answer]
The commit is just to move the test files.
I did not change anything. PhpStorm just did refactoring.

Writing return type is a good practice.
